### PR TITLE
Update Oct meeting / Jeff Fischer / IoT

### PR DESCRIPTION
**Pull request by glenjarvis**
Update October Meeting and make default

### CODE REVIEW

File: content/meetings/Meeting-2016-09-22.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ Thursday, September 22nd, 2016 Meeting
 :slug: meeting-2016-09-22
 :authors: Guido van Rossum
 :summary: Guido van Rossum - Python Language
-:save_as: index.html
+:save_as: meeting-2016-09.22.html
```
Comment (bdbaddog) [inline content/meetings/Meeting-2016-09-22.rst:11]: Better to just delete the save_as for any but the index page.

**Comment by bdbaddog** [comment]
I'll merge the pull and change the save_as and repush to same url and send you both an email to verify.

**Comment by glenjarvis** [comment]
I'll make a new PR to take out this save_as. I see it has a slug already, so I assume that is sufficient.

**Comment by bdbaddog** [comment]
Fixed already.
-Bill

**Pull request merged**